exchanges/btse/btse_test.go

```diff
@@ -597,95 +424,76 @@ func TestSubmitOrder(t *testing.T) {
 		AssetType: asset.Spot,
 	}
 	response, err := b.SubmitOrder(context.Background(), orderSubmission)
-	if sharedtestvalues.AreAPICredentialsSet(b) && (err != nil || response.Status != order.New) {
-		t.Errorf("Order failed to be placed: %v", err)
-	} else if !sharedtestvalues.AreAPICredentialsSet(b) && err == nil {
-		t.Error("Expecting an error when no keys are set")
-	}
+	assert.NoError(t, err, "SubmitOrder should not error")
+	assert.Equal(t, order.New, response.Status, "Response Status should be New")
 }
 
 func TestCancelAllAfter(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 
 	err := b.CancelAllAfter(context.Background(), 1)
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(t, err, "CancelAllAfter should not error")
 }
 
 func TestCancelExchangeOrder(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 
-	currencyPair := currency.NewPairWithDelimiter(currency.BTC.String(),
-		currency.USD.String(),
-		"-")
+	// TODO: Surely this cannot work
 	var orderCancellation = &order.Cancel{
 		OrderID:       "b334ecef-2b42-4998-b8a4-b6b14f6d2671",
 		WalletAddress: core.BitcoinDonationAddress,
 		AccountID:     "1",
-		Pair:          currencyPair,
+		Pair:          spotPair,
 		AssetType:     asset.Spot,
 	}
 	err := b.CancelOrder(context.Background(), orderCancellation)
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, err, "CancelOrder should not error")
 }
 
 func TestCancelOrder(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 
-	_, err := b.CancelExistingOrder(context.Background(), "", testSPOTPair, "")
-	if err != nil {
-		t.Fatal(err)
-	}
+	// TODO: Surely this cannot work
```

Comment:
For tests behind `SkipTestIfCredentialsUnset` with `canManipulateRealOrders`, we don't expect the test details to be accurate, the user should fill it themselves, if that's what you're `TODO`ing about. If that's the case, I think you should be removing the TODO

Reply:
Thanks for the clarification.
I afraid I don't agree that it's a reasonable expectation for contributors to change the order number, but even if it was, it's definitely not clear.
It's unusual to expect any manual changes to source code, including tests, to get things to work.
A .env file, a flag, or a env var are usual. I've never seen the likes of "fill in your creds / order number here" in Go code before.
In these cases, we should *make* the orders we expect to cancel, so that the test suites are self-fulfilling.
That's what the TODOS are for.
Now that you've clarified 459, I've change these TODO line to match: 3cda30fd

Reply:
Your views aside, your original todo was not going to get merged into `master` as it was not helpful for contributors and was more akin to a personal note. I'm glad that the context I gave you allowed you to improve your commentary

Reply:
Yep; Sorry, These comments as they stood shouldn't have been committed at all. I should have been clearer about that in my reply.
Thanks for catching them 🙏